Wins works

jbubar opened the pull request:
This is a collaborative PR between me and @akibrhast. He redesigned the structure of the wins page, and I put the finishing touches. @akibrhast put most of the work in for this pr. 
I also added some code for restructuring the card so that the quotes show up in the right place

Fixes #1066
closes #1060

akibrhast committed:
Implement filter on wins page

akibrhast committed:
Merge branch 'gh-pages' into filter-dropdown-wins-page#1030

akibrhast committed:
Add some media queries to resize filters

akibrhast committed:
Remove the css change made in the previous push cause it broke everything

akibrhast committed:
Team dropdown does not floats off the screen anymore

akibrhast committed:
Remove debug statements

akibrhast committed:
Reworked the filter css

akibrhast committed:
Fix css rule on wins which was breaking filters on project

akibrhast committed:
Made completed the functionality of the wins form

akibrhast committed:
fixed a missing file

akibrhast committed:
Update docker compose with increment tag

akibrhast committed:
wins page changes

akibrhast committed:
update wins

akibrhast committed:
Update code with comments

akibrhast committed:
Improve Validation

jbubar committed:
fix see more, restructuring cards, put in right order, and a bunch of other small things

jbubar committed:
delete commented code

jbubar committed:
fix merge conflict

jbubar commented:
<details>
<summary>desktop view</summary>

![screencapture-localhost-4000-wins-2021-02-28-09_59_21](https://user-images.githubusercontent.com/53061723/109428828-a0dbe200-79ad-11eb-8c8d-e3978f156c98.png)


</details>
<details>
<summary>Mobile</summary>

![screencapture-localhost-4000-wins-2021-02-28-10_00_13](https://user-images.githubusercontent.com/53061723/109428836-a6392c80-79ad-11eb-9e58-b344faf9c7a7.png)

</details>
<details>
<summary>overlay</summary>

![Screen Shot 2021-02-28 at 9 59 51 AM](https://user-images.githubusercontent.com/53061723/109428799-886bc780-79ad-11eb-9297-a00bff8ee79b.png)

</details>

jbubar committed:
Update docker-compose.yml

jbubar committed:
fixes docker-compose file?

jbubar committed:
Merge branch 'wins-works' of https://github.com/jbubar/website into wins-works

martham0 commented on _sass/components/_wins-page.scss:
```diff
@@ -251,42 +293,58 @@
 }
 
 .wins-card-content > * {
+<<<<<<< HEAD
```
Is this supposed to be like this? It looks like it's comparing the same css from the master branch and your own branch

ItsLaszlo reviewed (changes requested):
I could not run the branch. It keeps exiting with code 1 after running the command ``docker-compose up`` this seems to be due to some scss error.

![image](https://user-images.githubusercontent.com/69860036/109541463-b1a55a00-7a78-11eb-909f-a8cb1707ae66.png)

This might be because of this. I have never seen scss written this way it looks like it's comparing css from 2 different branches.

![image](https://user-images.githubusercontent.com/69860036/109542399-da7a1f00-7a79-11eb-9140-9d208e3a5309.png)

jbubar committed:
Update _wins-page.scss

Merge conflicts that slipped through the cracks

jbubar commented:
@martham0 Thanks for pointing that out. not sure how that slipped through the cracks. Next time I will be a lot more careful with resolving merge conflicts. 
Does that sort it out?

akibrhast commented on docker-compose.yml:
```diff
@@ -10,4 +10,4 @@ services:
       - 4000:4000
```
please remove the docke-compose.yml from this commit, since no changes are made in this file in relation to this pull request.

akibrhast reviewed (commented):
Everything works as expected! Ready to merge once docker-compose.yml has been removed from the commit. @jbubar

ItsLaszlo commented on _includes/wins-hero.html:
```diff
@@ -1,8 +1,11 @@
+
+    
     </div>
 </div>
```
Is this file still necessary? There seems to be another one with this exact code in ``wins/wins-hero.html``

akibrhast replied:
@martham0 Good spotting there!
@jbubar Could you please remove the file martha mentioned as well as the file in the root directory named `wins.html`

ItsLaszlo reviewed (commented):
> @martham0 Thanks for pointing that out. not sure how that slipped through the cracks. Next time I will be a lot more careful with resolving merge conflicts.
> Does that sort it out?

Yup! I am able to run it.

ItsLaszlo commented:
Everything seems to be working as expected! Only a quick note about a possible duplicate file mentioned above. 

This might be nitpicky and maybe a cause for another issue to be created but when using the dropdown filters the cursor change isn't accurate. It changes to ``cursor: pointer`` when hovering over anywhere in the dropdown except for the checkbox. On the checkbox, it turns back into a normal cursor. This can be a bit confusing for the user and implies they can select the category by clicking it rather than clicking the checkbox which isn't the case.

akibrhast commented:
> Everything seems to be working as expected! Only a quick note about a possible duplicate file mentioned above.
> 
> This might be nitpicky and maybe a cause for another issue to be created but when using the dropdown filters the cursor change isn't accurate. It changes to `cursor: pointer` when hovering over anywhere in the dropdown except for the checkbox. On the checkbox, it turns back into a normal cursor. This can be a bit confusing for the user and implies they can select the category by clicking it rather than clicking the checkbox which isn't the case.

@martham0 Could you please open a new ticket with what you mentioned here? The same behavior can also be seen on the projects page as well. I am guessing we could make a ticket that calls for an action to fix it in both places?

ItsLaszlo commented:
> > Everything seems to be working as expected! Only a quick note about a possible duplicate file mentioned above.
> > This might be nitpicky and maybe a cause for another issue to be created but when using the dropdown filters the cursor change isn't accurate. It changes to `cursor: pointer` when hovering over anywhere in the dropdown except for the checkbox. On the checkbox, it turns back into a normal cursor. This can be a bit confusing for the user and implies they can select the category by clicking it rather than clicking the checkbox which isn't the case.
> 
> @martham0 Could you please open a new ticket with what you mentioned here? The same behavior can also be seen on the projects page as well. I am guessing we could make a ticket that calls for an action to fix it in both places?

Will do! The issue can be found here: #1155

akibrhast commented:
blalbla
long image
<details>
<summary>this is broken</summary>

![image](https://user-images.githubusercontent.com/32349001/109746879-1f9a7000-7ba4-11eb-97c6-d3af19ea54b6.png)

</details>

akibrhast commented:
Hey @jbubar If I am not mistaken.. This pull request will also close
#1060 .
Could you confirm that, and if it is case , link it to this pull request

jbubar committed:
delete files

jbubar committed:
remove docker-compose from pr

jbubar commented:
> Hey @jbubar If I am not mistaken.. This pull request will also close
> #1060 .
> Could you confirm that, and if it is case , link it to this pull request

@akibrhast. Right! I did. and I will create a new issue for when it is done #1163

Thanks. and thanks for your patience. been moving around a bunch this week. Next week I will have some more free time. Best of luck this sunday :))

akibrhast committed:
Update wins-hero.html

Commented out the link to the new  form, since it is still in development and we are still using the google form

akibrhast reviewed (approved):
Everything looks good and ready for merge. @martham0 Thanks for creating that issue, and feel free to go ahead and merge it once you have had the time to review it yourself :D!

drubgrubby reviewed (commented):
This is really great work!  I love the new way of structuring complex pages.  

The only thing I notice is a small thing on the layout of the cards: The linkedIn and GitHub images seem to be much closer together than they are on the current site.  

Current site: 
<img width="679" alt="image" src="https://user-images.githubusercontent.com/68244054/110248596-d0976680-7f26-11eb-8454-bd0435126e46.png">

This PR: 
<img width="700" alt="image" src="https://user-images.githubusercontent.com/68244054/110248602-d7be7480-7f26-11eb-8bfd-274cdb541e6f.png">

This may be correct, but I didn't see any discussion of it so I'd like to have @daniellex0 sign off on it before we merge.

jbubar commented:
> The only thing I notice is a small thing on the layout of the cards: The linkedIn and GitHub images seem to be much closer together than they are on the current site.

good catch!

jbubar committed:
css fixes

jbubar committed:
Merge branch 'wins-works' of https://github.com/jbubar/website into wins-works

drubgrubby commented:
@jbubar - Thanks for making the change to the space between the linkedin and github images.  It looks good to me and ready to merge. 

@martham0 - It seems like this is ready to merge.  If you agree, then please go ahead and squash and merge.

ItsLaszlo commented:
Everything looks good to me. I'll squash and merge!
